core/utils/global.js

```diff
@@ -10,6 +10,10 @@
  */
 'use strict';
 
+/**
+ * Provides a reference to the global object.
+ * @namespace Blockly.utils.global
+ */
 goog.module('Blockly.utils.global');
 
 
```

Comment:
Looks like we need an alias here (and maybe bring the export in line with the prevailing style)